src/serial/normalize-newlines.ts

```diff
@@ -0,0 +1,3 @@
+const normalizeNewlines = (s: string): string => s.replace(/(?<!\r)\n|\r(?!\n)/g, '\r\n');
+
+export default normalizeNewlines;

```

Comment:
Is normalizing new lines required or more for consistency?

Reply:
Following the algorithm outlined in the [HTML spec](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#multipart-form-data).